InterfacePython/testCommon.py

```diff
@@ -39,6 +39,12 @@ def compareGrids(self, gridA, gridB, bTestRuleNames = True):
             return
 
         # load test points (canonical domain only), make sure to avoid grid points, e.g., 1/2, 1/4, etc.
+        if (gridA.getNumDimensions() == 1):
```

Comment:
I don't see where you're using these. If the grid has no loaded points, then evaluate and the points should never be referenced.

Reply:
For 1D grids, using this function without the change will return an error because `mX1` to `mX5` are needed in `aBatchPoints` (line 61), but are unintialized.

Reply:
`aBatchPoints` is used only if `getNumLoaded() > 0` on line `102`.

It is better to move the definition of `aBatchPoints` inside the if-statement on line `102`, rather than to defined unused variables.

Reply:
Changed. I moved the if-statement to before the first use of `mX1`, moved `aBatchPoints` to be in this if-block, and removed the previous 1D points.